etcdmgmt/etcd-mgmt.go

```diff
@@ -33,6 +33,9 @@ var (
 	ExecName = "etcd"
 	// Configuration directory for storing etcd configuration
 	ETCDConfDir = "/var/lib/glusterd"
+	// Etcd -data-dir location. This contains WAL files and log snapshots
+	// https://coreos.com/etcd/docs/latest/admin_guide.html#data-directory
+	ETCDDataDir = "/var/lib/etcd"
```

Comment:
If it is guaranteed that lifecycle of etcd daemons/processes are exclusively controlled by GD2 only, this should be straight-forward and not confusing. If not (user invokes systemctl or configures etcd to spawn at startup), I need some more clarification on this. Made the default set to `/var/lib/etcd` based on conventions used in etcd packaging as illustrated in this [systemd service file](https://github.com/coreos/etcd/blob/master/contrib/systemd/etcd.service)